src/vmprof_config.h

```diff
@@ -13,13 +13,15 @@
 #elif defined(__OpenBSD__)
 #define PC_FROM_UCONTEXT sc_rip
 #elif defined( __APPLE__)
-  #if ((ULONG_MAX) == (UINT_MAX))
-    #define PC_FROM_UCONTEXT uc_mcontext->__ss.__eip
-  #elif defined(__arm__)
-    #define PC_FROM_UCONTEXT uc_mcontext->__ss.__pc
-  #else
-    #define PC_FROM_UCONTEXT uc_mcontext->__ss.__rip
-  #endif
+    #if ((ULONG_MAX) == (UINT_MAX))
+        #define PC_FROM_UCONTEXT uc_mcontext->__ss.__eip
+    #else
+        #if defined(__arm__) || defined(__arm64__)
+            #define PC_FROM_UCONTEXT uc_mcontext->__ss.__pc
+        #else
+            #define PC_FROM_UCONTEXT uc_mcontext->__ss.__rip
+        #endif
+    #endif
```

Comment:
I think this would fix the formatting:

```suggestion
  #if ((ULONG_MAX) == (UINT_MAX))
    #define PC_FROM_UCONTEXT uc_mcontext->__ss.__eip
  #else
    #if defined(__arm__) || defined(__arm64__)
      #define PC_FROM_UCONTEXT uc_mcontext->__ss.__pc
    #else
      #define PC_FROM_UCONTEXT uc_mcontext->__ss.__rip
    #endif
  #endif
```

Reply:
ok, code has been formatted